File: Modul-2/Soal 3/PRAK203-2410817120019-NadyaRamadhani.py
#input output pertama
a, b, i, j, x, y = map(float, input().split())
hasil = (a-b) * (i/j) - (x+y)
print(f"{hasil:.3f}")

print("")

#input output kedua
a, b = map(float, input().split())
i, j = map(float, input().split())
x, y = map(float, input().split())
hasil = (a-b) * (i/j) - (x+y)
print(f"{hasil:.3f}")

File: Modul-2/Soal 3/PRAK203-2410817120019-NadyaRamadhani.c
#include <stdio.h>
int main(){
//input output pertama
float a, b, i, j, x, y;
float hasil;

printf("");
scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);
hasil = (a-b) * (i/j) - (x+y);
printf("%.3f", hasil);

//input output kedua
printf("\n\n");
printf("");
scanf("%f %f", &a, &b);
printf("");
scanf("%f %f", &i, &j);
printf("");
scanf("%f %f", &x, &y);
hasil = (a-b) * (i/j) - (x+y);
printf("%.3f", hasil);

return 0;
}